src/arizona_tpl_compile.erl

```diff
@@ -243,7 +243,7 @@ is_dtd(_) -> false.
 
 block_struct(Block, P, I, State) ->
     {Mod, Fun} = block_mod_fun(Block, State),
-    Tree = Mod:Fun(maps:get(args, Block, #{})),
+    Tree = apply(Mod, Fun, [maps:get(args, Block, #{})]),
```

Comment:
This is us tricking `elvis`. The fix would have to go deeper, but we can do that at a later time (this has basically no performance hit).